### ISSUE TITLE
automatically update api swagger on new api gateway document déploiement 

### ISSUE THREAD

**Issue by yannick-fernand** [awslabs/aws-api-gateway-developer-portal#481]
1 - exposed an api gateway api into developer portal 
2 - change the corresponding api documentation on api gateway and deploy it 
3 - go to developper portal and check the swagger 
   a - the swagger has not change 
   b - i have to go into admin panel et push the **update**  bouton to see my changed 

Is there a way to have the swagger to be automatically updated on new api gateway documentation publication ?

**Comment by alansammarone** [comment]
+1. It's very error prone to have to manually hit the Update button every time, would love a way to do that programmatically.

**Comment by tej-rana** [comment]
I agree but at what stage or interval do we want the updates to happen? Maybe an automatic/manual option first so that manual update is still available and then maybe an interval configuration that automatically does what the update button click is doing.
